### DISCUSSION TITLE
REDIS Auth

### DISCUSSION THREAD

**Discussion by franciscolopezc** [outline/outline#3661] [Self Hosting] [answered]
Hi everyone,

I am configuring outline with docker and I have configured for REDIS a password to secure it, how can I authenticate outline to connect to redis? I have tried with REDIS_PASSWORD=<password> in the docker.env, but it gives authentication error:

`ReplyError: NOAUTH Authentication required.
    at parseError (/opt/outline/node_modules/redis-parser/lib/parser.js:179:12)
    at parseType (/opt/outline/node_modules/redis-parser/lib/parser.js:302:14) {
  command: { name: 'info', args: [] }
}
`

I don't understand where the error could be

Thanks

**Comment by almereyda** [comment, marked as answer]
Please have a look at the `.env.sample`, more precisely the section about setting up the connection to Redis:

https://github.com/outline/outline/blob/9faa5dd01172d501e0467372fefdabe6b3186a0d/.env.sample#L20-L27

As of http://www.iana.org/assignments/uri-schemes/prov/redis you should be able to use the

```
Scheme syntax:
  Example: redis://user:secret@localhost:6379/0?foo=bar&qux=baz
```

**Reply by franciscolopezc** [reply]
This is just what I was looking for, I had not seen that it could also be configured this way, thank you very much.

**Reply by almereyda** [reply]
You are very welcome.
